vips/vip-292/bscmainnet.ts

```diff
@@ -0,0 +1,30 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
+
+const vip292 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-292",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: FAST_TRACK_TIMELOCK,
+        signature: "",
+        params: [],
+        value: "1",
+      },
```

Comment:
I think we should replace this command by the commands to set the new receive limits from Ethereum and opBNB, setting the 2% margin respect to the sending limits

Reply:
[ad78301](https://github.com/VenusProtocol/vips/pull/255/commits/ad78301dc961010f4e14e0d9e77d45dd7c331fa9)